src/helpers/section.ts

```diff
@@ -0,0 +1,62 @@
+import Cell from '../Interfaces/model/cell.interface';
+import SectionModel from '../Models/section.model';
+
+/**
+ * create a new section with no cells
+ * @param cells cells to add to section
+ */
+export const createSection = async (cells?: Cell[]) => {
+    return await SectionModel.create({ cells: cells ? cells : [] });
+};
+
+/**
+ * get the section with _id provided, null if not found
+ * @param id _id of the section
+ */
+export const getSection = async (id: string) => {
+    return await SectionModel.findById(id);
+};
+
+/**
+ * Add cell to section
+ * @param id id of section
+ * @param cell cell to add
+ */
+export const addCellToSection = async (id: string, cell: Cell) => {
+    return await SectionModel.update({ _id: id }, { $push: { cells: cell } });
```

Comment:
There should be a max limit on the number of cells a section can have. Add a check for that?